docs/reference/enable-gpu.rst

```diff
@@ -1,46 +1,153 @@
-Enable Nvidia GPUs
+.. _nvidia-gpu:
+
+Enable NVIDIA GPUs
 ==================
 
-To be able to run workloads on GPU with DSS, you need to have:
+The following section describes how to configure the DSS to utilize the host's
+NVIDIA GPU. This is achieved by configuring the underlying MicroK8s, that the
+DSS is relying for running the containerised workloads.
+
+Prerequisites
+^^^^^^^^^^^^^
+
+* Have run the :ref:`tutorial` tutorial
```

Comment:
Is this an actual prerequisite? or more like a recommendation? If the latter, then we should add it as a sentence after the intro paragraph, saying something like: "It is recommended you are familiar with DSS before enabling GPUs. See the :ref:`tutorial` for getting started."

Reply:
It's needed, so that users have installed microK8s and have the `dss` CLI

Reply:
ACK! Updated it in 8bbc3a0